File: backend/src/main/java/com/backend/domain/feedback/repository/FeedbackRepository.java
package com.backend.domain.feedback.repository;

import com.backend.domain.feedback.entity.Feedback;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.Optional;

public interface FeedbackRepository extends JpaRepository<Feedback,Long> {
    Optional<Feedback> findByAnswerId(Long answerId);
}

File: backend/src/main/java/com/backend/domain/feedback/repository/FeedbackRepository.kt
package com.backend.domain.feedback.repository

import com.backend.domain.feedback.entity.Feedback
import org.springframework.data.jpa.repository.JpaRepository
import java.util.*

interface FeedbackRepository : JpaRepository<Feedback, Long> {
    fun findByAnswerId(answerId: Long): Feedback?
}
